Makefile

```diff
@@ -153,10 +158,20 @@ endif
 
 ifneq ($(OS),macosx)
 EMBEDDED_LIBS += -lutil
+else
+
 endif
 
 endif # WITHPYTHON
 
+ifeq ($(SHOW_LD_LIBS),1)
+LD_FLAGS += -Wl,-t
```

Comment:
What is this feature?

Reply:
Diagnostic feature. Prints all binary artifacts that go into the link.